regression/cbmc-library/CreateThread1/main.c

```diff
@@ -0,0 +1,9 @@
+#include <windows.h>
```

Comment:
Minor nit pick - could we slightly tweak the directory naming for these test? having a number postfix with no obvious separator gets a bit confusing where the function being tests *also* has a numeric postfix (e.g. `_InterlockedAnd161` is the first test of  the `_InterlockedAnd16` function) - perhaps a naming scheme like `functionUnderTest-N` might be clearer? Or maybe even sub-directories, but thats probably more mechanics than we'd want to get into.

Reply:
Indeed, that was very confusing. I have now used a `-01` suffix instead of `1`.